uniflow/model/__init__.py

```diff
@@ -3,4 +3,4 @@
 # this register all possible model server into ModelServerFactory through
 # ModelServerFactory.register(cls.__name__, cls) in AbsModelServer
 # __init_subclass__
-from uniflow.model.server import *  # noqa: F401, F403
+from uniflow.model.model_server import *  # noqa: F401, F403
```

Comment:
model subfolder should not have its server/client class. model should be used by extract/transform flow's server/client interface.

Reply:
Updated with refactor commit